rules/do-you-create-an-online-itinerary/rule.md

```diff
@@ -17,15 +14,34 @@ related:
   - do-you-check-your-boarding-pass
   - do-you-take-advantage-of-business-rewards-programs
 redirects: []
-
+created: 2015-04-28T05:11:40.000Z
+archivedreason: null
+guid: 66c1fa86-8bea-4638-a4dc-15c3361292e8
 ---
-
 As well as sending details of bookings to the traveler's Outlook calendar, you should also use [tripit.com](https://www.tripit.com) to consolidate the details of bookings for flights, hotels, car rentals, etc.
 
 TripIt can make a single, easy-to-read online itinerary that can be viewed by anyone you choose to share the link with, making it easy to keep co-travelers, spouses, co-workers, and others in the loop.
 
+<div style="background-color: lightgrey; padding: 10px; border-radius: 5px;">
+
+ Adam Cogan loved TripIt and uses it every time he travels. He says:
+
+"TripIt tells you when there’s delay, a gate change, baggage info, or other problems...
+
+Plus, I share the flight with a bunch of people including those who are picking me up … so they get notified too"
+
+</div>
+
```

Comment:
```suggestion
::: greybox

 Adam Cogan loved TripIt and uses it every time he travels. He says:

"TripIt tells you when there’s delay, a gate change, baggage info, or other problems...

Plus, I share the flight with a bunch of people including those who are picking me up … so they get notified too"
:::
```

This will give the following effect. 

<img width="920" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/57518417/75f5ff62-f599-4517-a8c8-7591a2a63553">


Just makes it consistent with the rest of rules